src/plugin-slots/ExtendedProfileFieldsSlot/index.jsx

```diff
@@ -0,0 +1,36 @@
+import { PluginSlot } from '@openedx/frontend-plugin-framework';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { useCallback } from 'react';
+import { patchProfile } from '../../profile/data/services';
+import { fetchProfile } from '../../profile/data/actions';
+
+import SwitchContent from '../../profile/forms/elements/SwitchContent';
+import EmptyContent from '../../profile/forms/elements/EmptyContent';
+import EditableItemHeader from '../../profile/forms/elements/EditableItemHeader';
+
+const ExtendedProfileFieldsSlot = () => {
+  const dispatch = useDispatch();
+  const extendedProfileValues = useSelector((state) => state.profilePage.account.extendedProfile);
+
+  const pluginProps = {
+    refreshUserProfile: useCallback((username) => dispatch(fetchProfile(username)), [dispatch]),
+    updateUserProfile: patchProfile,
+    profileFieldValues: extendedProfileValues,
+    formComponents: {
+      SwitchContent,
+      EmptyContent,
+      EditableItemHeader,
+    },
+  };
```

Comment:
Overall I think these make sense to have as `pluginProps`, but it'd be helpful to have some documentation for them. I noticed they are missing from the `README` for the slot, so adding them there with some descriptive info would be perfect!